data/scripts.js

```diff
@@ -1860,7 +1864,7 @@ exports.BattleScripts = {
 					break;
 				}
 			}
-			if (moves.length === 4 && !counter.stab && !hasMove['metalburst'] && (counter['physicalpool'] || counter['specialpool'])) {
+			if (moves.length === 4 && !counter.stab && !hasMove['conversion'] && !hasMove['metalburst'] && (counter['physicalpool'] || counter['specialpool'])) {
```

Comment:
Maybe in `queryMoves` do something like this:

`if (move.priority === 0 && (hasAbility['Protean'] || moves.includes('conversion')) && !(moveid in NoStab)) counter['stab']++;`